PWGHF/Tasks/taskD0parametrizedPID.cxx

```diff
@@ -42,6 +42,11 @@ struct TaskD0parametrizedPIDMC {
   HistogramRegistry registry{
     "registry",
     {{"hMassGen", "2-prong candidates (generated); #it{p}_{T}; #it{y}", {HistType::kTH2F, {{150, 0., 30.}, {8, 0, 4.0}}}},
+     {"hGenFixCMSYNoEta", "2-prong candidates (generated); #it{p}_{T}", {HistType::kTH1F, {{150, 0., 30.}}}},
+     {"hGenPtDepYNoEta", "2-prong candidates (generated); #it{p}_{T}", {HistType::kTH1F, {{150, 0., 30.}}}},
+     {"hGenFixCMSYCMSEta", "2-prong candidates (generated); #it{p}_{T}", {HistType::kTH1F, {{150, 0., 30.}}}},
+     {"hGenPtDepYALICE2Eta", "2-prong candidates (generated); #it{p}_{T}", {HistType::kTH1F, {{150, 0., 30.}}}},
+     {"hGenPtDepYCMSEta", "2-prong candidates (generated); #it{p}_{T}", {HistType::kTH1F, {{150, 0., 30.}}}},
```

Comment:
The names of histograms are not clear enough to indicate what the are for. Some are even missing Pt.

Reply:
Histogram names are fixed now